pandas/core/frame.py

```diff
@@ -2813,8 +2813,12 @@ def to_parquet(
             ``io.parquet.engine`` is used. The default ``io.parquet.engine``
             behavior is to try 'pyarrow', falling back to 'fastparquet' if
             'pyarrow' is unavailable.
-        compression : {{'snappy', 'gzip', 'brotli', None}}, default 'snappy'
-            Name of the compression to use. Use ``None`` for no compression.
+        compression : {{'snappy', 'gzip', 'brotli', 'lz4', 'zstd', None}},
```

Comment:
```suggestion
        compression : {{'snappy', 'gzip', 'brotli', 'lz4', 'zstd', None}}, default 'snappy'
```

Then could you remove `default 'snappy'` from the next line?

Reply:
Sure.